integrated_channels/moodle/client.py

```diff
@@ -398,8 +410,9 @@ def delete_content_metadata(self, serialized_data):
             return rsp
         moodle_course_id = parsed_response['courses'][0]['id']
         params = {
-            'wsfunction': 'core_course_delete_courses',
-            'courseids[]': moodle_course_id
+            'wsfunction': 'core_course_update_courses',
+            'courses[0][id]': moodle_course_id,
+            'courses[0][visible]': 0  # hide a course rather than doing a true delete
```

Comment:
[Thought/Curious] what if we added an `_apply_delete_transformation` method to the [moodle exporter](https://github.com/openedx/edx-enterprise/blob/master/integrated_channels/moodle/exporters/content_metadata.py) similar to [what we do with SAP ](https://github.com/openedx/edx-enterprise/blob/master/integrated_channels/sap_success_factors/exporters/content_metadata.py#L50) such that we update the serialized metadata payload to reflect the `'courses[0][visible]': 0`?

Reply:
Then we could use that payload instead of building the params dict

Reply:
I just made the suggested changes in [latest commit](https://github.com/openedx/edx-enterprise/pull/1881/commits/7182b831646f3ff281e1131343b052259c33a4d4), is that all or something else needs to be done?